src/dbxdeploy/package/PackageDeployer.py

```diff
@@ -56,3 +107,7 @@ def __invoke(self, packageMetadata: PackageMetadata, whlContentReadyCallback: ca
     def __upload(self, content, whlPath: str):
         self.__logger.info(f'Uploading WHL package to {whlPath}')
         self.__packageUploader.upload(content, whlPath, overwrite=True)
+
+    def __checkOfflineDependencies(self, packageMetadata: PackageMetadata):
+        for dependency in packageMetadata.dependencies:
+            self.__localPathsResolver.getDependencyLocalPath(dependency)
```

Comment:
No return? If the purpose of the getDependencyLocalPath() method is just to check dependencies, is must have better name and does not need to return any path. Or?

Reply:
purpose of `getDependencyLocalPath` is to return local Path of dependency based on dependency name and version and if it does not exists it throws an exception. So we are just iterating hoping not to get exception. If no exception is thrown dependencies match.